contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -244,6 +246,7 @@ def create_perseus_exercise(ccnode, kolibrinode, exercise_data):
             preset_id=format_presets.EXERCISE,
             original_filename=filename,
             file_size=file_size,
+            uploaded_by_id=user_id,
```

Comment:
Ah, hmm.. I see why this is needed, but is there any way we can extract a reasonable default, either from the image Files, the AssessmentItem, or the ContentNode? We often manually run export from the command line for people

Reply:
Not sure what would be the best to use especially for collaborative channels- seems a like it could be frustrating if one user has to take the brunt of the channel size. One other option would be to just specify a `--user-id=<user id>` on the command line